dist/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Movies</title>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+
+<script defer src="runtime.bundle.js"></script><script defer src="index.bundle.js"></script></head>
+<body>
+  <header id="header">
+    <nav class="wrapper nav">
+      <span class="material-symbols-outlined hide-menu">
+        Menu
+      </span>
+      <div class="logo">
+        <p>Amazing movies</p>
+      </div>
+      <ul class="nav-list">
+        <li class="nav-items" showComponent="movie-section">Movie shows</li>
+        <li class="nav-items" showComponent="episode-section">Episodes</li>
+        <li class="nav-items" showComponent="contact-section">Contacts</li>
+      </ul>
+    </nav>
+    <div class="header-content wrapper">
```

Comment:
- [ ] Kindly dynamically add the item counter to the movie section on the navbar according to the requirement.
<img width="491" alt="image" src="https://user-images.githubusercontent.com/73854999/204089766-ea9aeac9-2c27-4ca1-a1e6-5ced1da91b50.png">